sequencer/src/l1_client.rs

```diff
@@ -87,17 +82,30 @@ impl Committable for L1BlockInfo {
     }
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Default)]
 pub struct L1ClientOptions {
-    url: Url,
+    url: Option<Url>,
     finalized_block_tag: BlockNumber,
     retry_delay: Duration,
 }
-
 impl L1ClientOptions {
+    pub fn new(url: Option<Url>) -> Self {
```

Comment:
Trying to shoe horn new things into `L1ClientOptions` may have over complicated it a bit. I plan on revisiting this along with `L1Client` instantiation.